src/Symfony/Component/Form/Tests/Extension/PasswordHasher/Type/PasswordTypePasswordHasherExtensionTest.php

```diff
@@ -77,6 +77,42 @@ public function testPasswordHashSuccess()
         $this->assertSame($user->getPassword(), $hashedPassword);
     }
 
+    public function testPasswordHashSuccessWitnEmptyData()
```

Comment:
`testPasswordHashSuccessWitnEmptyData`

should be:

`testPasswordHashSuccessWithEmptyData`

Reply:
good catch, but it's already fixed in the `6.2` branch :) (see d88d5b34f71718b9ebad0fbde895c96748af1d1f)